package.json

```diff
@@ -125,8 +125,8 @@
     "comment-json": "^4.2.3",
     "express": "^4.18.2",
     "hamburgers": "^1.2.1",
-    "i18next": "^21.10.0",
-    "i18next-http-backend": "^1.4.5",
+    "i18next": "^23.7.6",
+    "i18next-http-backend": "^2.4.1",
     "i18next-resources-to-backend": "^1.1.2",
```

Comment:
Let's bump also `i18next-resources-to-backend`.
I can see nearly no changes in the changelog (i.e. commit history of their library) in between 1.1 and 1.2(latest), but let's bump it to satisfy the compliance rules of using possible latest versions.

See https://github.com/i18next/i18next-resources-to-backend/commits/main/